okapi/locale/ro_RO/LC_MESSAGES/ro_RO.po

```diff
@@ -0,0 +1,432 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: OKAPI\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2016-03-10 19:07+0200\n"
+"PO-Revision-Date: 2016-03-10 19:08+0200\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: ro_RO\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Poedit-KeywordsList: _;gettext;gettext_noop\n"
+"X-Poedit-Basepath: ../../../..\n"
+"Plural-Forms: nplurals=3; plural=(n==1?0:(((n%100>19)||((n%100==0)&&(n!=0)))?"
+"2:1));\n"
+"X-Generator: Poedit 1.8.7\n"
+"X-Poedit-SearchPath-0: .\n"
+"X-Poedit-SearchPathExcluded-0: .git\n"
+
+#: okapi/services/caches/formatters/gpx.php:389
+msgid ""
+"<b>Geocache coordinates have been changed.</b> They have been replaced with "
+"your own custom coordinates which you have provided for this geocache."
+msgstr ""
+"<b>Coordonatele geocutiei au fost modificate.</b> Acestea au fost înlocuite "
+"cu coordonate alese de tine pentru această geocutie."
+
+#: okapi/services/caches/formatters/gpx.php:395
+msgid ""
+"<b>Geocache coordinates have been changed.</b> Currently they point to one "
+"of the alternate waypoints originally described as:"
+msgstr ""
+"<b>Coordonatele acestei geocutii au fost modificate.</b> Acestea indică unul "
+"din punctele suplimentare descris ca:"
+
+#: okapi/services/caches/formatters/gpx.php:408
+msgid "Original geocache location"
+msgstr "Coordonatele iniţiale ale geocutiei"
+
+#: okapi/services/caches/formatters/gpx.php:410
+#, php-format
+msgid "Original (owner-supplied) location of the %s geocache"
+msgstr ""
+"Coordonatele originale (aşa cum au fost publicate de proprietar) ale "
+"geocutiei %s"
+
+#: okapi/services/caches/formatters/gpxfile.tpl.php:38
+#: okapi/services/caches/formatters/gpxfile.tpl.php:73
+msgid "hidden by"
+msgstr "ascunsă de"
+
+#: okapi/services/caches/formatters/gpxfile.tpl.php:75
+#, php-format
+msgid "%d recommendation"
+msgid_plural "%d recommendations"
+msgstr[0] "o recomandare"
+msgstr[1] "%d recomandări"
+msgstr[2] "%d recomandări"
+
+#: okapi/services/caches/formatters/gpxfile.tpl.php:76
+#, php-format
+msgid "found %d time"
+msgid_plural "found %d times"
+msgstr[0] "găsită o singură dată"
+msgstr[1] "găsită de %d ori"
+msgstr[2] "găsită de %d ori"
+
+# Generic term for trackable items (like geocoins and travelbugs).
+# and geokrets?
```

Comment:
I have assumed the generic "trackables" of all kinds. 
The romanian translation, for lack of a better term, is "travelling objects", which I also used in c:geo translations. 
Should we come up with a better term, I'll update all translations.

So this is ok.